utils/checkDiamondStorage.js

```diff
@@ -0,0 +1,23 @@
+import { ethers } from "hardhat";
+import diamondContract from "../../packages/contracts/src/dollar/Diamond.sol";
+
+async function main() {
+  const diamondStorageSlot = ethers.utils.keccak256("diamond.standard.diamond.storage");
+
+  // Deployed Diamond contract instance
+  const diamond = await diamondContract.deployed();
```

Comment:
a better solution would be to use [forge inspect](https://book.getfoundry.sh/reference/forge/forge-inspect) instead of using hardhat + js